Bindings/Java/Matlab/examples/c3dExport.m

```diff
@@ -1,3 +1,17 @@
+% Specify point used for calculating output joint reaction moments
+% 0 = electrical center, 1 = COP
+%outputMomentsPoint = 0;
```

Comment:
Consider making this a string input (e.g., 'electrical_center' and 'center_or_pressure') or rename the variable to reflect the boolean (e.g., `useCenterOfPressureAsMomentsPoint`).